src/frontend/src/tables/sales/SalesOrderTable.tsx

```diff
@@ -90,7 +90,7 @@ export function SalesOrderTable({
     return [
       {
         accessor: 'reference',
-        title: t`Sales Order`,
+        // title: t`Sales Order`,
```

Comment:
Question: Ist this left in intentionally?

Reply:
Fixed!